### ISSUE TITLE
Problem rendering images with CMYK colorspace

### ISSUE THREAD

**Issue by garnieretienne** [Kozea/WeasyPrint#315]
Problem:

When using images with the CMYK colorspace, the color are changed (like a dark background).

Sample image (from google image):

![Sample CMYK](http://towerofbabel.free.fr/test/SLW_LOGO_CMYK.jpg)

Sample HTML:

``` html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Test CMYK</title>
    <style>
      @page {
        size: 6in 9in;
        margin: 0;
      }

      .image {
        max-width: 100%;
        max-height: 100%;
      }
    </style>
  </head>
  <body>
    <img class="image" src="http://towerofbabel.free.fr/test/SLW_LOGO_CMYK.jpg" alt="CMYK Test Image" />
  </body>
</html>
```

Sample command:

`weasyprint test.html test.pdf`

Result: 

![html_pdf](https://cloud.githubusercontent.com/assets/223963/15295695/ff5f0f52-1b91-11e6-86d1-bdaa58f1d157.jpg)

System:

Ubuntu 14.04 / WeasyPrint version 0.27
Edit: Ubuntu 16.04 / Weasyprint version 0.28 affected too

Anyone having this problem too?

**Comment by liZe** [comment]
I can reproduce and the problem appears with all my PDF readers, but it doesn't appear when you generate a PNG. It looks like a problem in the way Cairo embeds CMYK images in PDF files.

**Comment by garnieretienne** [comment]
Converting to RGB colorspace works too, (`convert cmyk.jpg -colorspace RGB cmyk2.jpg`) but I'm not sure I can modify our user images (our PDF files are provided to print services).

**Comment by liZe** [comment]
It was a bug in Cairo and it's been fix fixed in [1.15.4](https://www.cairographics.org/releases/ChangeLog.cairo-1.15.4) (commit b207a932a2d3740984319dffd58a0791580597cd).
